### DISCUSSION TITLE
Medium Mod Filter

### DISCUSSION THREAD

**Discussion by Arcadius2006** [LBPUnion/ProjectLighthouse#708] [Ideas]
This is an idea for a Medium setting for mod filter

it should
1. block hashed lbp1 .gmat files
2. let users upload .tex, .mol, .anim, .gmat, .mat and .bev files
3. block .ff files

**Comment by TorutheRedFox** [comment]
maybe add a safe validator for files that tries to look for deserialization-related errors to prevent game crashes caused by malformed files

could also make filetypes generated by the game itself safer too
